core/modules/admin_bar/admin_bar.inc

```diff
@@ -225,14 +225,16 @@ function admin_bar_tree_dynamic(array $expand_map) {
     // If contained in $expand_map, then this is a (first) parent, and we need
     // to store by the defined 'parent' path for later merging, as well as
     // provide the expansion map arguments to apply to the dynamic tree.
-    if (isset($expand_map[$link['path']])) {
-      $parent_path = $expand_map[$link['path']]['parent'];
-      $link['expand_map'] = $expand_map[$link['path']]['arguments'];
-    }
-    // Otherwise, just store this link keyed by its parent path; the expand_map
-    // is automatically derived from parent paths.
-    else {
-      $parent_path = $result[$link['plid']]['path'];
+    if ($mlid != 278) {
```

Comment:
`$link['type']` is available, but is `int`s looks like could be `mlid`s or `plid`s not sure